### DISCUSSION TITLE
Plugin page

### DISCUSSION THREAD

**Discussion by toshywoshy** [pretalx/pretalx#1208] [Ideas] [answered]
I create my first plugin for Pretalx, [pretalx-youtube](https://github.com/toshywoshy/pretalx-youtube), it is heavily based on [pretalx-vimeo](https://github.com/pretalx/pretalx-vimeo), making my work very easy.
I cannot add it to the wiki, however I think we should have a file PLUGINS.md, which  is a list of all working plugins, someplace it is easy to list all plugins

**Comment by rixx** [comment, marked as answer]
The wiki plugin page should be editable by anybody with a GitHub account – and I just updated it to reflect the current plugins available.

I don't want a `PLUGINS.md` (or rather `PLUGINS.rst`), because that would place the burden on maintainers to a) make sure plugins get added, b) make sure plugins are still useful and up to date, c) will make it seem like we endorse or audit plugins, which we can't do.

**Reply by toshywoshy** [reply]
My apologies, it seems that the web interface does allow making updates through the web interface, it seems that my concept of forking and create a PR for the wiki does not work. I just added my pretalx-youtube plugin.

As for the PLUGINS.rst/md I understand that creating a plugin testing framework is a lot of work and most likely beyond the scope.
Maybe the plugin metadata should have a field that indicates tested on pretalx version x.x.x, while this is just metadata, it gives the consumer of the plugin an idea which version the writer made it for and how likely it is to still work.

**Reply by rixx** [reply]
Having plugin compatibility information available would indeed be a good feature, and definitely something to be placed in a machine-readable way in the plugin's app config, not in a human-readable text file.
